-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-32737/20884

ERROR: Build errors found during clang-tidy run.
```
DQMServices/Core/src/MonitorElement.cc:907:24: error: out-of-line definition of 'setStatOverflows' does not match any declaration in 'dqm::impl::MonitorElement' [clang-diagnostic-error]
  void MonitorElement::setStatOverflows(bool value) {
                       ^
Suppressed 1377 warnings (1376 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/DQMServices/Core/interface/MonitorElement.h
+++ b/DQMServices/Core/interface/MonitorElement.h
@@ -427,7 +427,7 @@ namespace dqm::impl {
     virtual void setCanExtend(unsigned int value);
     // We should decide if we support this (or make it default)
     DQM_DEPRECATED
-    virtual void setStatOverflows(unsigned int value);
+    virtual void setStatOverflows(bool value);
 
     // these should be non-const, since they are potentially not thread-safe
     virtual TObject const *getRootObject() const;

